tests/unit/components/test_hostblock.py

```diff
@@ -143,7 +142,7 @@ def assert_urls(
 
 def blocklist_to_url(filename):
     """Get an example.com-URL with the given filename as path."""
-    assert not os.path.isabs(filename), filename
+    assert not pathlib.Path(filename).is_absolute(), filename
```

Comment:
Instead of making this a `pathlib.Path` here, it should take a path object as an argument directly. Perhaps also rename the argument to `path`. Then `create_zipfile` and `create_blocklist` will need to be changed instead, so that they return a `pathlib.Path` object as well.